### PR TITLE
Nada/FEQ-2405/chore: unit tests for advert create, update, delete

### PR DESCRIPTION
**Pull request by nada-deriv**
Screenshot 2024-06-25 at 5 24 08 PM
Screenshot 2024-06-25 at 5 28 27 PM
Screenshot 2024-06-25 at 5 37 51 PM

### CODE REVIEW

File: src/hooks/api/advert/p2p-advert/useAdvertCreate.ts [hunk 1]
```diff
@@ -33,24 +33,16 @@ const useAdvertCreate = () => {
...
         return {
             ...data,
-            /** Indicates if this is block trade advert or not. */
-            block_trade: Boolean(data?.block_trade),
-            /** The advert creation time in epoch. */
-            created_time: data?.created_time ? new Date(data?.created_time) : undefined,
-            /** The activation status of the advert. */
-            is_active: Boolean(data?.is_active),
-            /** Indicates that this advert will appear on the main advert list. */
-            is_visible: Boolean(data?.is_visible),
```
Comment (ameerul-deriv) [inline src/hooks/api/advert/p2p-advert/useAdvertCreate.ts:48]: why delete these values? are they not being used?

File: src/hooks/api/advert/p2p-advert/useAdvertDelete.ts [hunk 2]
```diff
@@ -36,26 +36,16 @@ const useAdvertDelete = () => {
...
         return {
-            ...p2p_advert_update,
-            /** The activation status of the advert. */
-            is_active: Boolean(p2p_advert_update.is_active),
-            /** Indicates if this is block trade advert or not. */
-            is_block_trade: Boolean(p2p_advert_update.block_trade),
-            /** Indicates that the advert has been deleted. */
-            is_deleted: Boolean(p2p_advert_update.deleted),
-            /** Indicates that this advert will appear on the main advert list. */
-            is_visible: Boolean(p2p_advert_update.is_visible),
```
Comment (ameerul-deriv) [inline src/hooks/api/advert/p2p-advert/useAdvertDelete.ts:53]: same here

File: src/hooks/api/advert/p2p-advert/useAdvertUpdate.ts [hunk 3]
```diff
@@ -29,25 +29,16 @@ const useAdvertUpdate = () => {
...
         return {
-            ...p2p_advert_update,
-            /** The activation status of the advert. */
-            is_active: Boolean(p2p_advert_update.is_active),
-            /** Indicates if this is block trade advert or not. */
-            is_block_trade: Boolean(p2p_advert_update.block_trade),
-            /** Indicates that the advert has been deleted. */
-            is_deleted: Boolean(p2p_advert_update.deleted),
-            /** Indicates that this advert will appear on the main advert list. */
-            is_visible: Boolean(p2p_advert_update.is_visible),
```
Comment (ameerul-deriv) [inline src/hooks/api/advert/p2p-advert/useAdvertUpdate.ts:45]: and here

File: src/hooks/api/advert/p2p-advert/useAdvertCreate.ts (hunk 1 above)
Comment (nada-deriv) [reply src/hooks/api/advert/p2p-advert/useAdvertCreate.ts:48]: no.. they are not being used

**Review by ameerul-deriv — APPROVED** [review]

**Review by shayan-deriv — APPROVED** [review]

**Pull request merged**